wpilib/wpilib/drive/differentialdrive.py

```diff
@@ -95,7 +126,6 @@ def __init__(self, leftMotor, rightMotor):
 
     def arcadeDrive(self, xSpeed, zRotation, squaredInputs=True):
         """Arcade drive method for differential drive platform.
-        The calculated values will be squared to decrease sensitivity at low speeds
```

Comment:
What are you doing?

Reply:
This isn't accurate if you pass in `squaredInputs=False`.

Reply:
Make a PR upstream?

Reply:
Upstream doesn't have this for `arcadeDrive(xSpeed, zRotation, squaredInputs)`.

Reply:
Oh it's just for the default...I see